### DISCUSSION TITLE
[Issue]SP Modernization Scanner Tool Results don't return SPD 2010 WFs with manual triggers

### DISCUSSION THREAD

**Discussion by samGakhar** [pnp/modernization#548] [Q&A] [answered]
Hi All,       
The reason to open this thread is that we are facing a strange issue with SP modernization scanner tool results, I hope this community would be able to help me out with the resolution for the same.
       **Background:-** We are in process of migrating our SP online tenant existing SP2010 SPD WFs to power automate as   2010 WFs are retiring soon.  As per Microsoft recommendation, we ran the SharePoint Modernization Scanner tool to scan our tenants for legacy workflows usage. However, there is a discrepancy in the scanner tool results and the actual 2010 WFs count on the SharePoint site.        
**Result Comparison:-**  SharePoint site 2010 WFs count are not matching with the result count of modernization scanner tool results. 
Below is the example for one of our site, for your reference:-       
Scanner tool results in one of the sites:-       
2010 SPD WFs count as per the scanner tool result is 6.    
SharePoint site actual 2010 WFs count as shown by SharePoint designer too is 8      
**Issues Details:-** After initial analysis, we found out that the SP modernization scanner tool results don't return the SPD 2010 WFs having manual trigger as start option.

Looking for the resolution of the same. If any of the community members have faced the same issue in past or can give some pointers to resolve the same will be appreciated.

Thanks,
Sameer
--

**Comment by jansenbe** [comment, marked as answer]
Use version 2.18 to scan + be aware that workflows added on content types are not listed again on each list where that content type is used as that would result in a lot of duplicates. We however do list those (as duplicates) when list  was explicitly set to use content types
